src/applications/vaos/appointment-list/components/RequestedAppointmentsList.jsx

```diff
@@ -0,0 +1,107 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import LoadingIndicator from '@department-of-veterans-affairs/formation-react/LoadingIndicator';
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+import recordEvent from 'platform/monitoring/record-event';
+import * as actions from '../redux/actions';
+import {
+  selectFeatureRequests,
+  selectIsCernerOnlyPatient,
+} from '../../redux/selectors';
+import { selectPendingAppointments } from '../redux/selectors';
+import { FETCH_STATUS, GA_PREFIX } from '../../utils/constants';
+import { getVAAppointmentLocationId } from '../../services/appointment';
+import RequestListItem from './AppointmentsPage/RequestListItem';
+import NoAppointments from './NoAppointments';
+
+function RequestedAppointmentsList({
+  showScheduleButton,
+  isCernerOnlyPatient,
+  pendingAppointments,
+  pendingStatus,
+  facilityData,
+  fetchFutureAppointments,
+  startNewAppointmentFlow,
+}) {
+  useEffect(
+    () => {
+      if (pendingStatus === FETCH_STATUS.notStarted) {
+        fetchFutureAppointments();
```

Comment:
A future enhancement might be to just fetch requests, rather than requests and future appts.

Reply:
Added new `fetchPendingAppointment` action